packages/lem/lem.2022-12-10/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "Lem Devs <cl-lem-dev@lists.cam.ac.uk>"
+authors: [
+  "Dominic Mulligan"
+  "Francesco Zappa Nardelli"
+  "Gabriel Kerneis"
+  "Kathy Gray"
+  "Peter Boehm"
+  "Peter Sewell"
+  "Scott Owens"
+  "Thomas Tuerk"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Thomas Bauereiss"
+  "Stephen Kell"
+  "Thomas Williams"
+  "Lars Hupel"
+  "Basile Clement"
+]
+homepage: "http://www.cl.cam.ac.uk/~pes20/lem/"
+bug-reports: "https://github.com/rems-project/lem/issues"
+license: "part BSD3, part LGPL 2"
```

Comment:
@kit-ty-kate sorry to bother you about this, I can't remember what is the right way to write this in the SPDX way

Reply:
I've fixed this, and also corrected the out of date LGPL version.